autoscheduler/frontend/src/components/SchedulingPage/ConfigureCard/ConfigureCard.tsx

```diff
@@ -117,6 +129,17 @@ const ConfigureCard: React.FC = () => {
             : 'Generate Schedules'}
         </Button>
       </div>
+      <Snackbar
+        open={showSnackbar}
+        autoHideDuration={10000}
```

Comment:
Like Gannon said in the meeting, we'll probably want to change this to ~5-6 seconds